### DISCUSSION TITLE
how to fix the error "Error: current directory is not an iraf user login directory"

### DISCUSSION THREAD

**Discussion by JxnuLin** [iraf-community/iraf#139] [Q&A] [answered]
when I run the command $mkiraf in my /iraf/iraf folder, I meet the error "Error: current directory is not an iraf user login directory". How to fix the error?

**Comment by olebole** [comment, marked as answer]
You should not run the command in the iraf directory, but in the directory that you want to use as the start for your iraf session(s).
